A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_LateralRevisionPage.js

```diff
@@ -11,35 +11,95 @@ class CDULateralRevisionPage {
         let coordinates = "---";
         console.log(waypoint);
         if (waypoint && waypoint.infos && waypoint.infos.coordinates) {
-            coordinates = waypoint.infos.coordinates.toDegreeString();
+            const lat = CDUInitPage.ConvertDDToDMS(waypoint.infos.coordinates['lat'], false);
```

Comment:
I'm not super happy about pulling this from the Init page - both the Init and the InitIRS page use this so far. Any suggestion for a more centralized place where this could live in future?

Reply:
`A320_Neo_CDU_MainDisplay.js`? Should be able to call it anywhere after with `mcdu.ConvertDDToDMS()`.

Reply:
Hm, it can be static though, and that might be more convenient for other places. Some kind of static helper class might be more useful. I'm just leaving this where it is now and we can pull it later if we need it in even more places.